### ISSUE TITLE
paymentSubtotals type / scientific notation in Invoice 

### ISSUE THREAD

**Issue by DorskFR** [bitpay/python-bitpay-client#110]
Hello,

Testing with `6.0.2` and an invoice of amount ~860 USD / 0.014057 BTC

https://github.com/bitpay/python-bitpay-client/blob/d2b084a43b9a852c57c6ce6ea36de6ddf6aba6e0/src/bitpay/models/invoice/invoice.py#L44

```
2 validation errors for Invoice
paymentSubtotals.MATIC
Unable to parse input string as an integer, exceeded maximum size [type=int_parsing_size, input_value=1.336520646e+21, input_type=float]
For further information visit https://errors.pydantic.dev/2.3/v/int_parsing_size
```

```python
invoice_data["paymentSubtotals"]
{
    "BTC": 1405700,
    "BCH": 202357900,
    "ETH": 301027000000000000,
    "DOGE": 590189044000,
    "LTC": 1099451000,
    "MATIC": 1.336520646e21,
}
```

**Comment by bobbrodie** [comment]
Hi @DorskFR -- thanks for reporting this! I see that's what is coming back from the API after creation, so before making a change I'm checking with a few folks on this to get a consensus on if we want to handle that in the SDK or the API. We'll report back shortly.

**Comment by bobbrodie** [comment]
@DorskFR quick update on this -- it's intentional in the API since these `int`s have their precision baked in. We'll write a patch to handle the case for scientific notation. Thanks again!

**Comment by bobbrodie** [comment]
This has been released as part of 6.0.3. Thanks again!
